explorer/file_test.cpp

```diff
@@ -55,11 +30,15 @@ class ParseAndExecuteTestFile : public FileTestBase {
       return false;
     }
 
+    bool check_trace_output =
+        path().string().find("/trace_testdata/") != std::string::npos;
```

Comment:
What does `check_trace_output` mean, and why is it different from `FLAGS_trace`?

Reply:
Trace output is only checked for a few tests. By default, even with FLAGS_trace, it is not checked. Also, FLAGS_trace is used independent of /trace_testdata/.